client/views/windows/connect.tpl

```diff
@@ -40,7 +40,13 @@
 			<div class="col-sm-9 col-sm-offset-3">
 				<label class="tls">
 					<input type="checkbox" name="tls" {{#if defaults.tls}}checked{{/if}} {{#if lockNetwork}}disabled{{/if}}>
-					Enable TLS/SSL
+					Enable TLS
+				</label>
+			</div>
+			<div class="col-sm-9 col-sm-offset-3">
+				<label class="tls">
+					<input type="checkbox" name="rejectUnauthorized" {{#if defaults.rejectUnauthorized}}checked{{/if}} {{#if lockNetwork}}disabled{{/if}}>
+					Verify TLS certificate trust chain
```

Comment:
tbh this raises even more questions to me 😅 

Going through the Node doc, I would call it any of: `(Accept|Allow)
 (untrusted|unverified|unauthorized) [TLS] certificate[s]` (I'm not sure "TLS" brings a lot of info when making the decision of checking/unchecking the checkbox).

That would reverse the meaning of checking/unchecking though (yours: checked is secure, unchecked is insecure; mine: checked is insecure, unchecked is secure) and I understand you'd rather keep the current state, so a contrapositive could be: `Accept (trusted|verified|authorized) [TLS] certificate[s] only`.

What do you think?

Reply:
Reworded it to `Allow untrusted certificates`.